src/primitives/types/types_typed.ts

```diff
@@ -121,18 +144,60 @@ export type TypeGenerators = {
   object: any;
   timestamp: any;
 
-  vec2: any;
```

Comment:
This is technically a breaking change, at least for TS users, I suppose.

When I initially added TS definitions, I only added full definitions for some properties, because I was still trying to understand the codebase (and still am, I suppose). I decided to leave untyped stuff as `any` rather than `unknown`, which would have been the safer but more annoying approach.

Anyway, improvements to static typing are always going to be somewhat breaking, so I don't think this warrants a major version bump.

Reply:
I'm fine with that, we can go semver once we get the README tidied up with all of this extra dev info and with updates on the changes we've made to the project.